core/src/com/unciv/ui/tilegroups/TileGroup.kt

```diff
@@ -285,11 +286,31 @@ open class TileGroup(var tileInfo: TileInfo, var tileSetStrings:TileSetStrings)
 
     fun isViewable(viewingCiv: CivilizationInfo) = showEntireMap
             || viewingCiv.viewableTiles.contains(tileInfo)
+            || !UncivGame.Current.worldScreen.fogOfWar
```

Comment:
Why is this needed? If fog of war is disabled, Isn't the viewingCiv the Speectator civ that can see everything anyway?
And if a check is needed then again to avoid the UncivGame.Current reference, I would check viewingCiv.isSpectator()

Reply:
You are right, viewingCiv.isSpectator() will be enough here.

Also wanted to comment that is was really great decision of yours to add viewingCiv as an argument to updates. That really made fog of war implementation very easy.